apps/condo/pages/employee/[id]/update.tsx

```diff
@@ -43,3 +52,20 @@ export const EmployeeUpdatePage = () => {
 EmployeeUpdatePage.requiredAccess = EmployeesReadAndManagePermissionRequired
 
 export default EmployeeUpdatePage
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+    const { req, res } = context
+
+    // @ts-ignore In Next 9 the types (only!) do not match the expected types
```

Comment:
Can we figure out, what to do? Seems this will be in like 20 apps on every page :(

Reply:
We can try to override the `GetServerSidePropsContext` type in the declaration files at app level

Reply:
I tried to override the types of the necessary types from NextJS, but it didn't work :( 
For some reason they are still used the original ones